commons/src/main/java/com/powsybl/commons/reporter/Report.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.commons.reporter;
+
+import com.powsybl.commons.PowsyblException;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ */
+public class Report {
+
+    public static final String REPORT_SEVERITY_KEY = "reportSeverity";
```

Comment:
why is this in the values instead of a normal field ?

Reply:
Severity might not be of interest for all consumers, hence using the values allows to keep the API generic.

Reply:
What about a default severity of "ALL" ? This way consummers who don't care about severity get everything and can ignore this part of the API, and others get what they want ?